### PR TITLE
Website crawler fix

### PR DESCRIPTION
**Pull request by ofermend**
Update to usage of playwright to address issues with crawling.

### CODE REVIEW

File: core/indexer.py [hunk 1]
```diff
@@ -274,7 +274,7 @@ def index_url(self, url: str, metadata: Dict[str, Any]) -> bool:
         # for everything else, use PlayWright as we may want it to render JS on the page before reading the content
         else:
             try:
-                actual_url, html_content = self.fetch_content_with_timeout(url)
+                actual_url, html_content = self.fetch_content_with_timeout(add_www(url))
```
Comment (justinhayes) [inline core/indexer.py]: This appears to always force a 'www' onto a URL. But it's normal for URLs to not have www (e.g. https://docs.vectara.com/). Is this going to break those URLs?

File: core/crawler.py [hunk 2]
```diff
@@ -17,35 +17,31 @@
...
-        session = requests.Session()
 
     # For binary files - we don't extract links from them, nor are they included in the crawled URLs list
-    # for document files (like PPT, DOCX, etc) we don't extract links from the, but they ARE included in the crawled URLs list
     url_without_fragment = url.split("#")[0]
     if any([url_without_fragment.endswith(ext) for ext in binary_extensions]):
         return visited
     visited.add(url)
+
+    # for document files (like PPT, DOCX, etc) we don't extract links from the, but they ARE included in the crawled URLs list
```
Comment (justinhayes) [inline core/crawler.py]: Typo in the comment - "... don't extract links from **the,** but ..."

**Comment by justinhayes** [comment]
Added a few comments. Other than that it looks good to me (although I am not very familiar with the playwright stuff).

File: core/indexer.py (hunk 1 above)
Comment (ofermend) [reply core/indexer.py]: I think you were looking at an older branch somehow? This is no longer in the code.

**Pull request merged**